packages/reporter/src/main.tsx

```diff
@@ -18,7 +18,7 @@ import shortcuts from './lib/shortcuts'
 import Header, { ReporterHeaderProps } from './header/header'
 import Runnables from './runnables/runnables'
 import TestingPreferences from './preferences/testing-preferences'
-import { BaseStore } from '@packages/runner-shared/src/store'
+import type { MobxRunnerStore } from '@packages/app/src/store/mobx-runner-store'
```

Comment:
Does this create a circular dependency?

Reply:
It does not - `@packages/app` does not import from `@packages/reporter`. Even if it did, this would not be a problem since we are doing `import type` which just grabs the definition - it won't import an actual JS module. `import type` is a really nice addition to TS.